ganga/GangaCore/GPIDev/Lib/File/File.py

```diff
@@ -97,7 +97,7 @@ def __setattr__(self, attr, value):
 
 
     def _attribute_filter__set__(self, attribName, attribValue):
-        if attribName is 'name':
+        if attribName=='name':
```

Comment:
PEP8 mandates spaces around the `==` operator
```suggestion
        if attribName == 'name':
```